Reverse,.cpp

```diff
@@ -0,0 +1,34 @@
+#include<iostream>
+using namespace std;
+/*void swap(int arr[start],int arr[end]){

```

Comment:
remove all the commented code

Reply:
> change this file directory to leetcode directory!

This problem is not present in leetcode but present in CodeChef so adding in CodeChef directory